client/ayon_maya/plugins/publish/collect_look.py

```diff
@@ -344,7 +344,14 @@ def collect(self, instance):
             "attributes": attributes,
             "relationships": sets
         }
-
+        if instance.data["includeTextureReferenceObjects"]:
+            input_collections, tx_object_nodes = (
+                self.collect_texture_reference_object_inputs(instance)
+            )
+            instance.data["lookData"].update(
+                {"texture_connections": input_collections}
+            )
```

Comment:
No need for `.update()` call here, plus I'd refactor `texture_connections` to just `connections`.
```suggestion
            instance.data["lookData"]["connections"] = (
                input_collections
            )
```